### ISSUE TITLE
SC 111 -- Image missing alt text

### ISSUE THREAD

**Issue by damiansian** [newjersey/accessibility-pm#24]
## URL
https://demo.fileyourstatetaxes.org/en/questions/post-data-transfer

## Issue
The image on the no alt text attribute. This is required in HTML for the img element. Since the graphic is covered in the adjacent heading text, the image is decorative and should be coded as such. 

## Screen shot
![Image](https://github.com/user-attachments/assets/7ac6d748-e5bd-4f8a-9b0c-8b49e8443ca0)

## WCAG Success Criterion
[SC 1.1.1: Non-text Content (Level A)](https://w3.org/TR/WCAG21#non-text-content)

## Current
```html
<img 
   src="/assets/icons/check-ea2b0c873d95329741139de4e443b73020f54086737cde2da8f6c0b396c0d2fc.svg">
``` 

### Should be:
```html
<img 
   alt=""
   src="/assets/icons/check-ea2b0c873d95329741139de4e443b73020f54086737cde2da8f6c0b396c0d2fc.svg">
```

**Comment by damiansian** [comment]
Looks good. `alt=""` used. 

<img width="1359" alt="Graphic is decorative. " src="https://github.com/user-attachments/assets/94b4f0a8-c97d-437b-8fa0-f0bb978133e0" />
